packages/design-system/src/components/EmptyState/EmptyState.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { EmptyStateLarge, EmptyStateMedium, EmptyStateSmall } from '.';
+
+export default {
+	component: EmptyStateLarge,
+};
+
+export const Large = () => (
+	<EmptyStateLarge
+		title="No preparations yet."
+		docLinkURL="https://talend.com"
```

Comment:
if I can use a callback why just a link URL here?

Reply:
Because I don't want you to pick the text for the link.